Add doc dir for Keptn 0.16 (#1228)
keptn/keptn.github.io: content/docs/0.16.x/monitoring/dynatrace/uninstall/index.md

---
title: Uninstall
description: Uninstall
weight: 5
icon: setup
---

## Uninstall the Dynatrace Keptn integration

To uninstall the *dynatrace-service*, use the Helm CLI to delete the release of the service:

```console
helm delete -n keptn dynatrace-service
```

> Note, this command only removes the Dynatrace integration into Keptn. Other components, such as the Dynatrace OneAgent on Kubernetes will be unaffected.

Reviewer: ```suggestion
Note, this command only removes the Dynatrace integration into Keptn. Other components, such as the Dynatrace OneAgent on Kubernetes are unaffected.
```
